mergify_cli/__init__.py

```diff
@@ -246,21 +247,30 @@ async def get_changeids_to_delete(
 async def create_or_update_comments(
     client: httpx.AsyncClient, pulls: list[PullRequest]
 ) -> None:
-    first_line = "This pull request is part of a stack:\n"
-    body = first_line
     for pull in pulls:
-        body += f"1. {pull['title']} ([#{pull['number']}]({pull['html_url']}))\n"
+        new_body = stack_comment_body(pulls, pull)
 
-    for pull in pulls:
         r = await client.get(f"issues/{pull['number']}/comments")
         check_for_status(r)
         for comment in r.json():
-            if comment["body"].startswith(first_line):
-                if comment["body"] != body:
-                    await client.patch(comment["url"], json={"body": body})
+            if comment["body"].startswith(STACK_COMMENT_FIRST_LINE):
+                if comment["body"] != new_body:
+                    await client.patch(comment["url"], json={"body": new_body})
                 break
         else:
-            await client.post(f"issues/{pull['number']}/comments", json={"body": body})
+            await client.post(
+                f"issues/{pull['number']}/comments", json={"body": new_body}
+            )
+
+
+def stack_comment_body(pulls: list[PullRequest], current_pull: PullRequest) -> str:
+    body = STACK_COMMENT_FIRST_LINE
+    for pull in pulls:
+        body += f"1. {pull['title']} ([#{pull['number']}]({pull['html_url']}))"
```

Comment:
We must increment the index here right? Looks like a regression here

Reply:
It's Markdown format syntax. It will be render correctly on GitHub.